test/e2e/test_job_submissions.py

```diff
@@ -95,7 +95,10 @@ def test_queue_credentials_file_is_secure_from_other_users(
             def is_job_started(current_job: Job) -> bool:
                 current_job.refresh_job_info(client=deadline_client)
                 LOG.info(f"Waiting for job {current_job.id} to be created")
-                return current_job.lifecycle_status != "CREATE_IN_PROGRESS"
+                return (
+                    current_job.lifecycle_status != "CREATE_IN_PROGRESS"
+                    and current_job.task_run_status in [TaskStatus.RUNNING, TaskStatus.SUCCEEDED]
```

Comment:
I think in this test, we want to run SSM commands while the session is running, right? With that in mind, should the test raise an exception if the `task_run_status` is in any status that we don't want? Looking at the [API docs](https://docs.aws.amazon.com/deadline-cloud/latest/APIReference/API_GetJob.html#deadlinecloud-GetJob-response-taskRunStatus), I'm thinking we want to fail if the job is in:

* `PENDING`
* `INTERRUPTING`
* `SUSPENDED`
* `CANCELED`
* `FAILED`
* `SUCCEEDED`
* `NOT_COMPATIBLE`

Same applies to the other `is_job_started` function below.

Reply:
If the session is in any of the other states, we would fail at this `is_job_Started` step because it would never be in the state that we want. But if we fail if the state is in `PENDING` for example, then that means that we aren't even waiting for the job to be started?

Reply:
Okay, makes sense, I'll publish a revision